search/management/commands/reindex_solr.py

```diff
@@ -34,19 +33,11 @@ class Command(BaseCommand):
     help = 'Take all sounds moderated and processed as OK and send them to Solr'
 
     def handle(self, *args, **options):
-        slice_size = 1000
-        num_sounds = Sound.objects.filter(processing_state="OK", moderation_state="OK").count()
-        for i in range(0, num_sounds, slice_size):
-            console_logger.info("Adding %i sounds to solr, slice %i", slice_size, i)
-            try:
-                # Get all sounds moderated and processed ok
-                where = "sound.moderation_state = 'OK' AND sound.processing_state = 'OK' AND sound.id > %s"
-                order_by = "sound.id ASC"
-                sounds_qs = Sound.objects.bulk_query_solr(where, order_by, slice_size, (i, ))
-                add_sounds_to_solr(sounds_qs)
-            except SolrException as e:
-                console_logger.error("failed to add sound batch to solr index, reason: %s", str(e))
-                raise
+        # Get all sounds moderated and processed ok
+        sounds_to_index = Sound.objects.filter(processing_state="OK", moderation_state="OK")
+        console_logger.info("Reindexing %d sounds to solr", sounds_to_index.count())
+
+        add_all_sounds_to_solr(sounds_to_index)
```

Comment:
I'd be good to add an option to the command to mark all sounds as index clean and make it default to `True`. Pass this option to `add_all_sounds_to_solr` (see comment above) so that sounds get marked as clean.